src/views/AuthoritiesSearch/ReportsModal/ReportsModal.js

```diff
@@ -0,0 +1,103 @@
+import { useIntl } from 'react-intl';
+import {
+  Field,
+  useFormState,
+} from 'react-final-form';
+import PropTypes from 'prop-types';
+
+import {
+  Button,
+  Modal,
+  ModalFooter,
+  DateRangeWrapper,
+  Datepicker,
+} from '@folio/stripes/components';
+import stripesFinalForm from '@folio/stripes-final-form';
+
+import { REPORT_TYPES } from '../constants';
+
+import styles from './ReportsModal.css';
+
+const propTypes = {
+  handleSubmit: PropTypes.func.isRequired,
+  onClose: PropTypes.func.isRequired,
+  open: PropTypes.bool.isRequired,
+  reportType: PropTypes.oneOf(Object.values(REPORT_TYPES)).isRequired,

```

Comment:
This is not required, the component is rendered even if `open` is false.

Reply:
The component is rendered, but it returns `null` if `open` is false

Reply:
With null, we'll see the warning like `reportType is marked as required, but its value is null`